suite/tests/CMakeLists.txt

```diff
@@ -3279,6 +3279,16 @@ endif ()
         "../../clients/drdisas/test_simple.template"
         "-thumb" "-mode;thumb;${drdisas_args}" ON)
     endif ()
+    # Test intel syntax
+    if (X86 OR AARCH64)
```

Comment:
So no ARM -- I assume b/c you don't have a machine to test it on?  Please add a comment to make it clear there's no technical reason this can't be on ARM.

Reply:
Done